### DISCUSSION TITLE
Maven Dependency Lookup Warnings with parent in same repository

### DISCUSSION THREAD

**Discussion by derkoe** [renovatebot/renovate#18669] [Request Help]
With Maven, a multi-module project usually has a parent `pom.xml` in the repository with all the modules. When this parent module is not published in a Maven repository Renovate shows a "Dependency Lookup Warning".

For the repository https://github.com/porscheinformatik/angular-spring-heroes the error is:

### ⚠ Dependency Lookup Warnings ⚠

-   Renovate failed to look up the following dependencies: `Failed to look up maven dependency at.porscheinformatik.angular-spring-heroes:heroes-parent`.

Files affected: `heroes-backend/pom.xml`, `heroes-frontend/pom.xml`, `heroes-webapp/pom.xml`

Here is the dependency dashboard: https://github.com/porscheinformatik/angular-spring-heroes/issues/250

Is there a workaround / solution for this?

**Comment by rarkins** [comment]
@zharinov this looks similar to some recent ones

**Reply by zharinov** [reply]
Yes, can be tracked in #18117
